pkg/autoscaler/multiscaler.go

```diff
@@ -168,7 +168,15 @@ func (m *MultiScaler) createScaler(ctx context.Context, kpa *kpa.PodAutoscaler)
 	}
 
 	stopCh := make(chan struct{})
-	runner := &scalerRunner{scaler: scaler, latestScale: -1, stopCh: stopCh}
+	runner := &scalerRunner{scaler: scaler, stopCh: stopCh}
+	now := time.Now()
+	stat := Stat{
+		Time: &now,
+		PodName: "mock-pod",
+		AverageConcurrentRequests: 1,
+		RequestCount: 1,
+	}
+	scaler.Record(ctx, stat)
```

Comment:
Recording a fake pod will keep the revision alive for the stable window after being deployed. It will not make sure it's not scaled down before anything is setup properly though.